src/lambdas/postprocessor/src/lambda_function.py

```diff
@@ -130,6 +107,11 @@ def handler(event, context):
         raise Exception("One or more jobs failed. See logs for details.")
 
     if rw_area_jobs and GLOBALS.env == "production":
-        # update AOIs on RW but only on production
-        geostore_ids = get_aoi_geostore_ids(rw_area_jobs[0].features_1x1)
-        update_area_statuses(geostore_ids, "saved")
+        try:
```

Comment:
This is something that only happens in the postprocessor, so I think we'd still want to potentially notify slack of this error? (although tbh I don't think it's ever had an error before)

Reply:
Good point, now notifying.